integration-test/tests/login-form.macos.spec.js

```diff
@@ -236,7 +236,7 @@ test.describe('Auto-fill a login form on macOS', () => {
     })
     test.describe('When availableInputTypes API is available', () => {
         test.describe('and I have saved credentials', () => {
-            test('I should be able to use my saved credentials by clicking', async ({page}) => {
+            test('I should be able to use my saved credentials by clicking @flaky', async ({page}) => {
```

Comment:
marking as flaky for now, since it was observed as such locally (before this change). after a number of CI runs we can decide if these changes have helped, or not :)